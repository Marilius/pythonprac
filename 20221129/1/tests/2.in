class C(metaclass=dump):
    def __init__(self, *a, **b):
        self.a = a
        self.b = b

    def add(self, *a, **b):
        self.a += a
        self.b.update(b)
        return self.a, self.b


c = C(10, amogus='ы')
print(c.add(9, рыба='золото'))
print(c.add(9, курячий='жизнь'))