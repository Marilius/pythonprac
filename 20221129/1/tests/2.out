__init__: (10,), {'amogus': 'ы'}
add: (9,), {'рыба': 'золото'}
((10, 9), {'amogus': 'ы', 'рыба': 'золото'})
add: (9,), {'курячий': 'жизнь'}
((10, 9, 9), {'amogus': 'ы', 'рыба': 'золото', 'курячий': 'жизнь'})